static/app/components/modals/metricWidgetViewerModal/header.tsx

```diff
@@ -8,49 +7,52 @@ import {IconCheckmark, IconEdit} from 'sentry/icons';
 import {space} from 'sentry/styles/space';
 import {WidgetDescription} from 'sentry/views/dashboards/widgetCard';
 
-type Props = {
-  displayValue: string;
-  placeholder: string;
-  value: string;
+export interface TitleState {
```

Comment:
I would prefer if we exported this under the domain it belong to, or be a bit more specific. `TitleState` seems overly generic and ends up polluting our global namespace. Fwiw, this is an issue across the entire codebase, so no need to fix it, as it wont actually improve much, but I would rather we put a bit of thought in the naming in general :)

Reply:
Makes sense! Thanks! I did at least prefix it with `Metrics` in [0a7d2fa](https://github.com/getsentry/sentry/pull/67104/commits/0a7d2faaad24efa10a23e01b7223adad70360765) to make it a bit more specific